generated/nidmm/nidmm.proto

```diff
@@ -229,6 +229,34 @@ enum CableCompensationType {
   CABLE_COMPENSATION_TYPE_NIDMM_VAL_CABLE_COMP_OPEN_AND_SHORT = 3;
 }
 
+enum CalibrationType {
+  option allow_alias = true;
+  CALIBRATION_TYPE_UNSPECIFIED = 0;
+  CALIBRATION_TYPE_NIDMM_VAL_INTERNAL_AREA = 0;
+  CALIBRATION_TYPE_NIDMM_VAL_EXTERNAL_AREA = 1;
+}
+
+enum CalibrationCloseAction {
+  option allow_alias = true;
+  CALIBRATION_CLOSE_ACTION_UNSPECIFIED = 0;
+  CALIBRATION_CLOSE_ACTION_NIDMM_EXTCAL_ACTION_ABORT = 0;
+  CALIBRATION_CLOSE_ACTION_NIDMM_EXTCAL_ACTION_SAVE = 1;
+}
+
+enum CompensatedOhms {
+  option allow_alias = true;
+  COMPENSATED_OHMS_UNSPECIFIED = 0;
+  COMPENSATED_OHMS_NIDMM_VAL_OFFSET_COMP_OHMS_OFF = 0;
+  COMPENSATED_OHMS_NIDMM_VAL_OFFSET_COMP_OHMS_ON = 1;
+}
+
+enum DCBias {
+  option allow_alias = true;
+  D_C_BIAS_UNSPECIFIED = 0;
```

Comment:
The `D_C_` should be `DC`. Same with `L_C_` below.

Reply:
this is autogenerated to D_C_. same is the case for the attributes that were already present with DC or LC names. should I change this by editing nidmm.proto manually?

Reply:
replaced DC with Dc so now DC is generated. done similar for LC, ADCC, and RTD which have similar generation